apps/fabric-website/src/pages/Customizations/ColorsCustomizationPage.tsx

```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
-import { PageHeader } from '../../../components/PageHeader/PageHeader';
-import { ThemeGeneratorPage } from '@uifabric/fabric-website-resources/lib/components/pages/ThemeGeneratorPage';
-const pageStyles: any = require('../../PageStyles.module.scss');
+import { PageHeader } from '../../components/PageHeader/PageHeader';
+import { ColorsPage } from '@uifabric/fabric-website-resources/lib/components/pages/ColorsPage';
```

Comment:
This is not a "Colors" page though, the page specifically generates themes and does nothing else, not even documentation.

Reply:
Hmm maybe so. Brainstorm in the morning on this.

We could have a page later whoch describes what palette is and what it contains.

Reply:
Agreed, there's already a page for Colors under styles. What about `ColorsCustomizationsPage`?

Reply:
OMG nevermind I didn't read down to the next line 😂 My bad.